src/Service.Tests/CosmosTests/TestBase.cs

```diff
@@ -46,7 +46,7 @@ public class TestBase
     id : ID!,
     name : String,
     character: Character,
-    age : Int,
+    age : Float,
```

Comment:
why age is not Int?

Reply:
That was Sourabh's choice to make in #2307 to test this behavior. When we make similar test for SQL via #2319 we don't have to go that route.